cmd/timestamp-cli/app/timestamp.go

```diff
@@ -37,15 +37,14 @@ import (
 	"github.com/spf13/viper"
 )
 
-func addTimestampFlags(cmd *cobra.Command) error {
```

Comment:
This function never calls any functions that could produce errors or creates an error itself, so I removed this return type.